foodgroup/chat_nav.go

```diff
@@ -24,6 +24,11 @@ var defaultExchangeCfg = wire.TLVBlock{
 	},
 }
 
+var ErrChatNavRoomNameMissing = errors.New("unable to find chat name in TLV payload")
```

Comment:
I think these errors should be unexported (i.e. start with a lower-case `e`). I can't see the calling code checking for these errors in particular.

also nit:

It's typical to group all errors in one block:

```
var (
    ErrChatNavRoomNameMissing = errors.New("unable to find chat name in TLV payload")
    ErrChatNavRoomCreateFailed = errors.New("unable to create chat room")
    ErrChatNavRetrieveFailed = errors.New("unable to retrieve chat room chat room")
    ErrChatNavMismatchedExchange = errors.New("chat room exchange does not match requested exchange")
)